src/util/VanDerCorput.cpp

```diff
@@ -116,10 +131,8 @@ VanDerCorput::const_iterator VanDerCorput::end() const
 }
 
 //=============================================================================
-VanDerCorput::const_iterator::const_iterator(const VanDerCorput *seq)
-    : mSeq(seq)
-    , mN(-1)
-    , mFinalIter(false)
+VanDerCorput::const_iterator::const_iterator(const VanDerCorput* seq)
+  : mSeq(seq), mN(-1), mFinalIter(false)
```

Comment:
I actually somewhat prefer the former, mostly because the trailing comma issue really bugs me.  I am somewhat ambivalent here as long as we are consistent.